### PR TITLE
add system setting for error.log which defaults to 'logs/' and uses this configurable path for logging

### PR DESCRIPTION
**Pull request by gerrit1986**
worked on this at modx ccc 2015. 
i hope it does not break anything. 
any feedback is appreciated.
https://github.com/modx-ccc-2015/whishlist/issues/39

### CODE REVIEW

**Comment by markwillis82** [comment]
I haven't tested this yet - but what would happen if you cannot connect to the database? Where would the error log?

File: core/model/modx/processors/system/errorlog/clear.class.php [hunk 1]
```diff
@@ -10,7 +10,8 @@ public function checkPermissions() {
         return $this->modx->hasPermission('error_log_erase');
     }
     public function process() {
-        $file = $this->modx->getOption(xPDO::OPT_CACHE_PATH).'logs/error.log';
+        $error_log_path = $this->modx->getOption('error_log_path');
+        $file = MODX_BASE_PATH.$error_log_path.'error.log';
```
Comment (goldsky) [inline core/model/modx/processors/system/errorlog/clear.class.php]: This should be an absolute path, eg `/home/website.tld/core/error.log`

**Comment by theboxer** [comment]
Closing due to long inactivity. Feel free to reopen with more details and rebased to 2.x branch

**Comment by Mark-H** [comment]
This was closed as the contributor did not respond to questions about the implementation.

**Pull request closed without merging**